integration-cli/docker_utils_test.go

```diff
@@ -263,13 +264,13 @@ func daemonTime(c *check.C) time.Time {
 	assert.NilError(c, err)
 
 	dt, err := time.Parse(time.RFC3339Nano, info.SystemTime)
-	c.Assert(err, check.IsNil, check.Commentf("invalid time format in GET /info response"))
+	assert.Assert(c, err == nil, "invalid time format in GET /info response")
```

Comment:
`assert.NilError(c, err, ...)` (we can probably reach if there's others like this)